kaldb/pom.xml

```diff
@@ -99,12 +99,6 @@
                 </exclusion>
             </exclusions>
         </dependency>
-        <dependency>
```

Comment:
We added this to the classpath to test `io_uring`, but are blocked on further unrelated infrastructure changes. Pulling out for now as this is an incubator dep, and we'll revisit once we're ready to try `io_uring` again.